File: app/src/main/java/br/com/sevenestoque/sevenestoque/ClienteAdapter.java
package br.com.sevenestoque.sevenestoque;

import android.content.Context;
import android.support.v7.widget.RecyclerView;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import java.util.List;

class ClienteAdapter extends RecyclerView.Adapter<ClienteAdapter.MyHoder> {

    List<FireModel> list;
    Context context;

    public ClienteAdapter(List<FireModel> list, Context context) {
        this.list = list;
        this.context = context;
    }

    @Override
    public MyHoder onCreateViewHolder(ViewGroup parent, int viewType) {

        View view = LayoutInflater.from(context).inflate(R.layout.item_cliente, parent, false);
        MyHoder myHoder = new MyHoder(view);

        return myHoder;
    }


    @Override
    public void onBindViewHolder(MyHoder holder, int position) {
        FireModel mylist = list.get(position);
        holder.name.setText(mylist.getNome());
        holder.CPF.setText(mylist.getCPF());
        holder.contato.setText(mylist.getContato());
        holder.email.setText(mylist.getEmail());
    }

    @Override
    public int getItemCount() {

        int arr = 0;

        try {
            if (list.size() == 0) {

                arr = 0;

            } else {

                arr = list.size();
            }


        } catch (Exception e) {


        }

        return arr;

    }

    class MyHoder extends RecyclerView.ViewHolder {
        TextView name, CPF, contato, email;


        public MyHoder(View itemView) {
            super(itemView);
            name = (TextView) itemView.findViewById(R.id.txtNome);
            CPF = (TextView) itemView.findViewById(R.id.txtCpf);
            contato = (TextView) itemView.findViewById(R.id.txtContato);
            email = (TextView) itemView.findViewById(R.id.txtEmail);

        }
    }

}

File: app/src/main/java/br/com/sevenestoque/sevenestoque/ClienteAdapter.kt
package br.com.sevenestoque.sevenestoque

import android.support.v7.widget.RecyclerView
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.ImageButton
import android.widget.TextView
import java.util.*

class ClienteAdapter(var lista:ArrayList<Cliente>):RecyclerView.Adapter<ClienteAdapter.ViewHolder> () {
    override fun onBindViewHolder(holder: ViewHolder, posicao: Int) {
        var cliente :Cliente = lista[posicao]
        holder?.cpf.text = cliente.cpf.toString()
        holder?.nome.text = cliente.nome.toString()
        holder?.contato.text = cliente.contato.toString()
        holder?.email.text = cliente.email.toString()

    }

    override fun getItemCount(): Int {
        return lista.size
    }

    override fun onCreateViewHolder(parent: ViewGroup, pl: Int): ViewHolder {
        val v:View = LayoutInflater.from(parent?.context).inflate(R.layout.item_cliente,parent,false)
        return ViewHolder(v)
    }

    class ViewHolder(itemview:View):RecyclerView.ViewHolder(itemview) {
        val cpf = itemview.findViewById<TextView>(R.id.txtCpf)
        val nome = itemview.findViewById<TextView>(R.id.txtNome)
        val contato = itemview.findViewById<TextView>(R.id.txtContato)
        val email = itemview.findViewById<TextView>(R.id.txtEmail)
        val btnEdit = itemview.findViewById<ImageButton>(R.id.ibtnEdit)
        val btnDelete = itemview.findViewById<ImageButton>(R.id.ibtnDelete)
    }




}